### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by asaganda**
Added FactoryGirl configuration details

### CODE REVIEW

**Comment by asaganda** [comment]
@vinvasir

File: README.md [hunk 1]
```diff
@@ -929,6 +929,31 @@ config.after(:suite) do
...
+
+**Configurating Tests (Factory Girl)**
+
+FactoryGirl setup for the model that's getting the Paperclip attachment:
+
+```ruby
+FactoryGirl.define do
+  factory :asset do
+    attachment { Rack::Test::UploadedFile.new("#path_to_attachment", "attachment_file_type (e.g. image/png)") }
+  end
```
Comment (tute) [inline README.md]: Thanks! What do you think of a less generic example, like:

```ruby
factory :user do
  avatar { File.new("#{Rails.root}/spec/support/fixtures/image.jpg") }
end
```

(taken from https://stackoverflow.com/questions/3294824/how-do-i-use-factory-girl-to-generate-a-paperclip-attachment)

File: README.md [hunk 2]
```diff
@@ -929,6 +929,31 @@ config.after(:suite) do
   FileUtils.rm_rf(Dir["#{Rails.root}/spec/test_files/"])
 end
 ```
+
+**Configurating Tests (Factory Girl)**
```
Comment (tute) [inline README.md]: I'd rephrase this to:

> **Configurating Tests with Factory Girl**

What do you think of that?

File: README.md [hunk 3]
```diff
@@ -929,6 +929,31 @@ config.after(:suite) do
...
+
+FactoryGirl setup for the model that's getting the Paperclip attachment:
```
Comment (tute) [inline README.md]: If we change the title, maybe we don't need this sentence.

File: README.md (hunk 2 above)
Comment (tute) [inline README.md]: Or:

> Example of test configuration with Factory Girl

File: README.md [hunk 4]
```diff
@@ -929,6 +929,31 @@ config.after(:suite) do
...
+
+```ruby
+RSpec.configure do |config|
+  # ...
+  config.after(:suite) do
+     FileUtils.rm_rf(Dir["#path_where_attachments_are_saved"])
+  end
+  #...
+end
+```
```
Comment (tute) [inline README.md]: Images will be loaded in `tmp/`, which is always safe to remove, and can stay as a cache. I don't think we'll need this cleanup section.

**Comment by tute** [comment]
Related issue: https://github.com/thoughtbot/paperclip/issues/1432

**Comment by tute** [comment]
Thank you. 🚀
